src/scripting/floating_image.cpp

```diff
@@ -28,8 +28,8 @@ FloatingImage::FloatingImage(const std::string& spritefile) :
 {
   using namespace worldmap;
 
-  if(Sector::current() != NULL) {
-    Sector::current()->add_object(floating_image);
+  if(::Sector::current() != NULL) {
```

Comment:
Is there a reason for this change?

Reply:
Floating_image was getting confused with the Sector scripting class and the actual Sector class.